Functions/Engines/Wine/script.js

```diff
@@ -358,6 +360,38 @@ Wine.prototype._installWinePackage = function (setupWizard, winePackage, localDi
         .extract();
 };
 
+Wine.prototype._installGecko = function (setupWizard, winePackage, localDirectory) {
+    var gecko = new Resource()
+        .wizard(setupWizard)
+        .url(winePackage.geckoUrl)
+        .checksum(winePackage.geckoMd5)
+        .algorithm("md5")
+        .name(winePackage.geckoFile)
+        .directory("gecko")
+        .get();
+
+    var wineGeckoDir = localDirectory + "/share/wine/gecko";
+    mkdir(wineGeckoDir);
+
+    lns(gecko, wineGeckoDir + "/" + winePackage.geckoFile);
+};
+
+Wine.prototype._installMono = function (setupWizard, winePackage, localDirectory) {
+    var mono = new Resource()
+        .wizard(setupWizard)
+        .url(winePackage.monoUrl)
+        .checksum(winePackage.monoMd5)
+        .algorithm("md5")
+        .name(winePackage.monoFile)
+        .directory("mono")
+        .get();
+
+    var wineMonoDir = localDirectory + "/share/wine/mono";
+    mkdir(wineMonoDir);
+
+    lns(mono, wineMonoDir + "/" + winePackage.monoFile);
```

Comment:
Yes so if you install any gecko package, it will update all wine versions. It will allow to fix the urls if required (and it is often required because we can be late sometimes)